File: LLM-Translations/GPT-4/codenet/C/Go/s683308527.go

package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	n, _ := strconv.Atoi(scanner.Text())

	for i := 0; i < n; i++ {
		scanner.Scan()
		s := scanner.Text()
		s = strings.Replace(s, "Hoshino", "Hoshina", -1)
		fmt.Println(s)
	}
}

// 

File: LLM-Translations/GPT-4/codenet/C/Java/s683308527.java
import java.util.Scanner;

public class s683308527 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();
        
        while (n-- > 0) {
            String s = sc.nextLine();
            s = s.replaceAll("Hoshino", "Hoshina");
            System.out.println(s);
        }
    }
}
// 
